### PR TITLE
feat: added seed cli command support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,33 +1,70 @@
...
+    nodejs npm
 
-RUN npm install
-COPY . .
-RUN npm run build
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+RUN apt-get install -y nodejs
 
-FROM node:12.20.1-alpine3.9 as production
+RUN node -v
```
Comment (asafMasa) [inline Dockerfile]: ?

File: README.md [hunk 2]
```diff
@@ -1,125 +1,2 @@
...
-```bash
-npm run test:unit
-```
-
-To only run integration tests:
-```bash
-npm run test:integration
-```
+# mapproxy-seed
+MapProxy seed deployment based on mapproxy-utils
```
Comment (asafMasa) [inline README.md:2]: readme

File: package.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "service-name",
+  "name": "mapproxy-seed",
   "version": "1.0.0",
   "description": "This is template for map colonies typescript service",
```
Comment (asafMasa) [inline package.json]: update the description

File: src/cliBuilder.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      },
+      'refresh-before': {
+        alias: 'rb',
+        describe: 'supported absolute datetime format: "yyyy-MM-ddTHH:mm:ss", refresh tile before this datetime',
+        type: 'string',
+        demandOption: true,
+      },
+      'skip-uncached': {
+        alias: 's',
+        describe: 'skip uncached tiles, seeds or refresh only cached tiles',
```
Comment (asafMasa) [inline src/cliBuilder.ts]: add default behavior to description

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply Dockerfile]: removed

File: package.json (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply package.json]: fixed

File: src/cliBuilder.ts (hunk 4 above)
Comment (CL-SHLOMIKONCHA) [reply src/cliBuilder.ts]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**